index.html

```diff
@@ -2,7 +2,7 @@
 <html>
 <head>
   <title>
-DID Core Registry v1.0
+DID Data Model Registries v1.0
```

Comment:
```suggestion
DID Specification Registries v1.0
```

Suggested on Editors call.